ui/src/utils/timeSeriesToDygraph.js

```diff
@@ -163,10 +171,17 @@ export default function timeSeriesToDygraph(raw = [], isInDataExplorer) {
   }
 }
 
-export const timeSeriesToTable = data => {
-  const {labels, timeSeries} = timeSeriesToDygraph(data, false)
-  const tableData = timeSeries.length
-    ? timeSeries.map(row => row.map(cell => (cell ? cell.toString() : 'null')))
-    : [[]]
-  return {labels, data: [labels, ...tableData]}
+export const timeSeriesToTableGraph = raw => {
```

Comment:
Would like to see a test for this update to the `timeSeriesToDygraph` function

Reply:
I will do this as a part of the next PR I think we might be changing things around in this function.